lib/devise/multi_email/models/validatable.rb

```diff
@@ -7,7 +7,6 @@ module EmailValidatable
 
       included do
         validates_presence_of   :email, if: :email_required?
-        validates_uniqueness_of :email, allow_blank: true, if: :email_changed?
```

Comment:
@terryyin Sorry for the delay. One more question, why this uniqueness validation is removed?

Reply:
This is because in my case I don't store emails as plain text and `email` is not a real field.

Do you prefer to have it still? All tests pass without it.

Reply:
yep... I think we should have it. The ensures the newly added email are unique in DB.

I think I might missed the tests for it, hmm.